src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -24,7 +24,7 @@ class Blueprint
     /**
      * The columns that should be added to the table.
      *
-     * @var \Illuminate\Support\Fluent[]
+     * @var Columns\Column[]
```

Comment:
Please use fully qualified names in all phpdoc.

Reply:
Done